wagtail/admin/views/tags.py

```diff
@@ -1,7 +1,9 @@
+from django.conf import settings
 from django.contrib.contenttypes.models import ContentType
 from django.http import Http404, JsonResponse
 from taggit.models import Tag, TagBase
 
+WAGTAIL_TAGS_AUTOCOMPLETE_LIMIT = 10
```

Comment:
Do you mind updating this to 

```suggestion
TAGS_AUTOCOMPLETE_LIMIT = 10
```

Without the WAGTAIL_ prefix.

If we do add a setting usage in the future we can rename accordingly.

For now it's just an abstracted constant.

Also a reminder to remove the `settings` import as it's no longer used.

Reply:
Sure, I'll make the changes